lib/reek/smells/long_parameter_list.rb

```diff
@@ -43,7 +43,7 @@ def inspect(ctx)
           context: ctx,
           lines: [exp.line],
           message: "has #{count} parameters",
-          parameters: { count: count })]
+          parameters: { count: count, name: ctx.name })]
```

Comment:
Nope, we're calling `full_name` in the `smell_warning` method, so that's different. 
I was confused by this as well, but we should fix this separately.